docs/reference/commandline/build.md

```diff
@@ -31,7 +31,7 @@ parent = "smn_cli"
       --pull=false                    Always attempt to pull a newer version of the image
       -q, --quiet=false               Suppress the verbose output generated by the containers
       --rm=true                       Remove intermediate containers after a successful build
-      -t, --tag=""                    Repository name (and optionally a tag) for the image
+      -t, --tag=[]                    Repository names (and optionally with tags) to apply to the image
```

Comment:
@moxiegirl this line is from the output of `docker build --help` command.  Perhaps this help message is too long. How about omitting the second sentence?